### DISCUSSION TITLE
Account safety issue

### DISCUSSION THREAD

**Discussion by AptinYT** [pear-devs/pear-desktop#2245] [General]
Hello I have a question
Is this safe to login? Because I dont want to get hacked and scam because there's a lot information and my files document
![Screenshot 2024-07-16 152352](https://github.com/user-attachments/assets/4300c6c7-67cc-4f05-838d-d2fa97ad4d1e)

**Comment by JellyBrick** [comment]
Yes.

**Reply by AptinYT** [reply]
is it 100% save because i dont know about you i want admins to answer it and tick like fully answer

**Reply by ArjixWasTaken** [reply]
> is it 100% save because i dont know about you i want admins to answer it and tick like fully answer 

You did not make a discussion that can accept an answer, so we can't tick it as an answer.

Btw jellybrick is an admin

**Reply by AptinYT** [reply]
ok then i trust this
